css/style.css

```diff
@@ -104,6 +104,72 @@ body {
   padding-top: 2.2rem;
 }
 
+/* DETAIL PAGE  */
+
+.popup {
+  display: none;
+  flex-direction: column;
+  justify-content: center;
+  position: sticky;
+  background: #fff;
+  top: 0;
+  padding: 1rem 1rem;
+  margin: 2rem 0.8rem;
+  border-radius: 10px;
+  height: 100%;
+  z-index: 2;
```

Comment:
- [ ] Great job implementing the popUp on both mobile and desktop screen sizes :clinking_glasses: 
- [ ] However, I noticed that in the mobile view, the design is missing the blurred background as in the Figma design and this doesn't make your project looks so good since the background can still be seen, it interferes with the popup window so kindly fix this :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/216154190-efd33d39-ca3e-427d-b305-fa1b80432b05.png)

- Figma Design
![image](https://user-images.githubusercontent.com/102699958/216154114-7a74b8dd-4a25-4399-9aef-572028f57df7.png)